src/theme/style-overrides/components.ts

```diff
@@ -0,0 +1,16 @@
+import { ThemeOptions } from "@mui/material";
+
+const COMPONENTS_STYLE_OVERRIDES: ThemeOptions["components"] = {
+  MuiButton: {
+    styleOverrides: {
+      root: {
+        paddingInline: "16px",
+      },
+      outlinedPrimary: ({ theme }) => ({
+        border: `1px solid ${theme.palette.primary.main}`,
```

Comment:
Which bit is this changing — the colour?

Reply:
Yeah neither do I. But hopefully `unstable_sxConfig`
Unless I'm misunderstanding it
<img width="430" alt="Screenshot 2024-05-08 at 14 18 07" src="https://github.com/telicent-oss/telicent-ds/assets/153610820/df1c12cb-f9db-48f7-8c2f-ed411a354e6e">
— [source](https://mui.com/material-ui/customization/default-theme/)

Reply:
> Which bit is this changing — the colour?

The colour is not being changed here, the colours are being changed/modified in the palette section of the theme object. 

Here I am modifying the default style of the outline button to match what Justin has designed. The outline colour by default has a reduced opacity. But we don't have the reduced opacity.

Reply:
Ah. 

Idea: Eventually it might be useful to have:

```ts
expect(muiDefaultThemeOptions).toMatchSnapshot(telicentDarkThemeOptions);
```

Reply:
Yeah I think using the unstable_sxConfig might be a way to do it.